### PR TITLE
Set acceleration only once in falling node

### PR DESCRIPTION
**Pull request by Rui914**
And serialize node data.

### CODE REVIEW

File: builtin/game/falling.lua [hunk 1]
```diff
@@ -5,43 +5,43 @@
...
-			textures = {node.name},
-		}
-		self.object:set_properties(prop)
+			textures = {node.name}
+		})
 	end,
 
 	get_staticdata = function(self)
-		return self.node.name
+		return core.serialize(self.node)
```
Comment (est31) [inline builtin/game/falling.lua]: This will break old words, for what meaning? Better keep it the old way.

File: builtin/game/falling.lua [hunk 2]
```diff
@@ -5,43 +5,43 @@
...
 
 	set_node = function(self, node)
 		self.node = node
-		local prop = {
+		self.object:set_properties({
 			is_visible = true,
-			textures = {node.name},
-		}
-		self.object:set_properties(prop)
+			textures = {node.name}
```
Comment (est31) [inline builtin/game/falling.lua]: please put a comma at the end.

File: builtin/game/falling.lua [hunk 3]
```diff
@@ -5,43 +5,43 @@
...
-		collisionbox = {-0.5,-0.5,-0.5, 0.5,0.5,0.5},
-		visual = "wielditem",
-		textures = {},
-		visual_size = {x=0.667, y=0.667},
-	},
+	visual = "wielditem",
+	visual_size = {x = 0.667, y = 0.667},
+	textures = {},
+	physical = true,
+	is_visible = false,
```
Comment (est31) [inline builtin/game/falling.lua]: where does this come from?

**Comment by est31** [comment]
Otherwise looks good

File: builtin/game/falling.lua (hunk 3 above)
Comment (Rui914) [inline builtin/game/falling.lua]: >Deprecated: Everything in object properties is read directly from here

https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L3294

File: builtin/game/falling.lua (hunk 1 above)
Comment (Rui914) [inline builtin/game/falling.lua]: It should be core.serialize because self.node has param2 value.
Also I added supporting old staticdata (line 38).

Comment (est31) [inline builtin/game/falling.lua]: please fix this.

File: builtin/game/falling.lua (hunk 3 above)
Comment (est31) [inline builtin/game/falling.lua]: Aww seems you will have to keep it the old way. Please revert the changes done here directly, and look up in a dictionary what the english word "deprecated" means.

File: builtin/game/falling.lua (hunk 1 above)
Comment (est31) [inline builtin/game/falling.lua]: Ah okay then, its fine.

Comment (Rui914) [inline builtin/game/falling.lua]: But falling nodes doesn't rotate by param2.
Should I fix this?

Comment (est31) [inline builtin/game/falling.lua]: nono, sorry i haven't read your comment, what you do here is fine

**Comment by est31** [comment]
:+1: looks good

**Comment by paramat** [comment]
:+1:

File: builtin/game/falling.lua (hunk 1 above)
Comment (est31) [inline builtin/game/falling.lua]: @Rui914 if you want to fix it, go on, do it :)

File: builtin/game/falling.lua (hunk 3 above)
Comment (HybridDog) [inline builtin/game/falling.lua]: http://www.dict.cc/?s=deprecated

l think the deprecated refers to initial_properties because on_activate and on_step are nothing deprecated

Comment (est31) [inline builtin/game/falling.lua]: @HybridDog no, `initial_properties` is not deprecated, its the other way round: dc70f50b5901732f928c439060dcbbcc6a5dfeb7

**Comment by paramat** [comment]
65c09a96f41705bb8e75fc5ff4276342be91ed11
